Tools/CISupport/ews-build/config.json

```diff
@@ -217,74 +217,46 @@
       "workernames": ["ews191", "ews192", "ews193", "ews194", "ews195", "ews196", "ews197", "ews198"]
     },
     {
-      "name": "macOS-AppleSilicon-Sonoma-Debug-Build-EWS", "shortname": "mac-AS-debug", "icon": "buildOnly",
+      "name": "macOS-Sonoma-Debug-Build-EWS", "shortname": "mac-AS-debug", "icon": "buildOnly",
```

Comment:
Should consider removing AS from the shortname as well. Same thing for macOS-Sonoma-Debug-WK2-Tests-EWS queue.